Makefile

```diff
@@ -1499,7 +1499,7 @@ done
 if test -n "$$skip"; then
   ${call emphasise,Error: Cannot link objects to build $@:}
   ${call emphasise,Module $$skip is faulty.}
-  exit 1
+  rm -f .*.dis; exit 1
```

Comment:
I took the liberty of removing the newline here and in the following, so that `git grep exit` shows at a glance that they all have the corresponding `rm -f .*.dis`.

That way every `exit` implies a `rm` on the same point, and vice versa except for the one preceeding `${if ${VERB},echo "done."}` (I wasn't sure what you'd want to do with this one).